modules/sievefilters/setup.php

```diff
@@ -124,6 +129,8 @@
         'ajax_sieve_block_domain',
         'ajax_sieve_block_change_behaviour',
         'ajax_sieve_toggle_script_state',
+        'ajax_liste_block_sieve',
```

Comment:
liste -> list

Reply:
hello marc; I did not grasp the preoccupation. What is the request?

Reply:
liste is a word in French.

We want source code in English: https://en.wiktionary.org/wiki/list